docs/chapters/flow/index.md

```diff
@@ -332,16 +329,11 @@ metas:
   name: complete indexer
 ```
 
-This type of construction will act as a single `indexer` and will allow to seamlessly `query` this index with the `embedding` vector coming
-from any upstream `encoder` and obtain in the response message of the pod the corresponding `binary` information stored in
-the key-value index. This lets the `VectorIndexer` be responsible to obtain the most relevant documents by finding similarities
-in the `embedding` space while targeting the `key-value` database to extract the meaningful data and fields from those relevant documents.
+This acts as a single indexer, letting you seamlessly query the index with the embedding vector from any upstream encoder. It returns the binary information in the key-value index in the Pod's response message. This lets the `VectorIndexer` be responsible for getting the most relevant Documents by finding similarities in the embedding space while targeting the key-value database to extract meaningful data and fields from the relevant Documents.
```

Comment:
Totally my fault, but can we think of expressing this with shorter sentences? it is tiring to read right?

Reply:
Yeah, exhausting ;) I'll break it down more